### PR TITLE
[explorer] - top packages table

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

Add the Popular Packages to the home screen both for current / future layout

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/explorer/src/pages/home/Home.tsx [hunk 1]
```diff
@@ -93,10 +93,10 @@ function Home() {
                     </TabPanels>
                 </TabGroup>
             </div>
-            {/* TODO: Add the popular packages component here :) */}
-            {/* <div style={{ gridArea: 'packages' }} className="mt-5 bg-gray-60">
-                Popular packages
-            </div> */}
+
+            <div style={{ gridArea: 'packages' }}>
+                <TopPackagesCard />
```
Comment (mamos-mysten) [inline apps/explorer/src/pages/home/Home.tsx:98]: one callout here:

I was having issues nesting two TabGroups (one for the Packages table header, and the other for the date filter) as it was inheriting styling from the parent tab group and marking everything in the date filter as selected. 

so in order to not have to hoist the filter state up to the home page I dropped the components down a level into `TopPackagesCard`

Comment (Jordan-Mysten) [reply apps/explorer/src/pages/home/Home.tsx:98]: Yeah tbh datefilter being tabs is really wrong, technically. It should be a radio button. That's my bad.

File: apps/explorer/src/components/top-packages/TopPackagesCard.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+export function TopPackagesCard() {
+    const rpc = useEnhancedRpcClient();
+    const [selectedFilter, setSelectedFilter] = useState<DateFilter>('3D');
+
+    const { data, isLoading } = useQuery(
+        ['top-packages', selectedFilter],
+        async () => rpc.getMoveCallMetrics()
+    );
+
+    const filteredData = useMemo(() => {
```
Comment (williamrobertson13) [inline apps/explorer/src/components/top-packages/TopPackagesCard.tsx:31]: does this need to be memo-ized?

File: apps/explorer/src/pages/home/Home.tsx [hunk 3]
```diff
@@ -93,10 +93,10 @@ function Home() {
                     </TabPanels>
                 </TabGroup>
             </div>
-            {/* TODO: Add the popular packages component here :) */}
-            {/* <div style={{ gridArea: 'packages' }} className="mt-5 bg-gray-60">
-                Popular packages
-            </div> */}
+
```
Comment (williamrobertson13) [inline apps/explorer/src/pages/home/Home.tsx:96]: could we make sure this looks good on all breakpoints for the new design?

**Review by williamrobertson13 — APPROVED** [review]
lgtm

File: apps/explorer/src/components/top-packages/TopPackagesCard.tsx (hunk 2 above)
Comment (mamos-mysten) [reply apps/explorer/src/components/top-packages/TopPackagesCard.tsx:31]: not really, just carried it over from some previous code. I can remove.

File: apps/explorer/src/pages/home/Home.tsx (hunk 3 above)
Comment (mamos-mysten) [reply apps/explorer/src/pages/home/Home.tsx:96]: double checked, looks good

File: apps/explorer/src/pages/home/Home.tsx (hunk 1 above)
Comment (mamos-mysten) [reply apps/explorer/src/pages/home/Home.tsx:98]: Created a JIRA to cover the follow up work

File: apps/explorer/src/pages/home/Home.tsx [hunk 4]
```diff
@@ -5,7 +5,6 @@ import clsx from 'clsx';
 import { lazy, Suspense } from 'react';
 
 import { ErrorBoundary } from '../../components/error-boundary/ErrorBoundary';
-// import { RecentModulesCard } from '../../components/recent-packages-card/RecentPackagesCard';
```
[the remark this answers is not in the document]
Comment (mamos-mysten) [reply apps/explorer/src/pages/home/Home.tsx:8]: yesssir, deleted.

**Pull request merged**